lib/Client.js

```diff
@@ -2181,15 +2182,10 @@ class Client extends EventEmitter {
         if(status) {
             this.presence.status = status;
         }
-        if(activities === null) {
-            activities = [];
-        } else if(activities && !Array.isArray(activities)) {
+        if(activities && !Array.isArray(activities)) {
             activities = [activities];
         }
-        if(activities !== undefined) {
-            this.presence.activities = activities;
-        }
-
+        this.presence.activities = activities || [];
```

Comment:
If `activities` is undefined, this should just be left alone. There shouldn't be a change in activities. 

TLDR: previous handling of `null` should continue having that behaviour. Your change would also be breaking

The PR should just add support for custom status

Reply:
Tfw the TLDR is longer than the original message
I reverted it back👍🏻